pom.xml

```diff
@@ -57,6 +57,22 @@
 			<version>18.0</version>
 		</dependency>
 		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-core</artifactId>
+			<version>1.3</version>
+		</dependency>
+		<dependency>
+			<groupId>net.java.dev.jna</groupId>
+			<artifactId>jna</artifactId>
+			<version>4.1.0</version>
```

Comment:
Relooking at this this morning, what seems to have happened.. is that my intellij did not see the src/it/java as "test" source so it would not allow me to run the code without removing the <scope>test</scope>.... but just marking that dir as test also fixes it... will add the <scope>test</scope> back.

![image](https://cloud.githubusercontent.com/assets/654946/6884249/9c51a802-d5e7-11e4-807e-2bf8f78953e9.png)